Automattic/jetpack
# IDC: deprecate unused methods

This is the second part of the IDC package cleanup. After deleting previously deprecated methods, this PR is deprecating other unused methods. 

## Proposed changes:

*

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Create a JN site and use debug tools to trigger the Safe Mode. Everything should work.
* Check the debug log. If testing on a sandbox, there should be at least one deprecation warning about the dynamic URL.
